### PR TITLE
added libffi as a depend of cicada-shell

### PR DESCRIPTION
**Pull request by mitnk**
Hi @flipee, Thanks for making this package available on AUR. When I trying it out (with an Arch docker), I found makepkg failed because of missing `libffi`. I guess it should be added as a new depend?

### CODE REVIEW

**Comment by flipee** [comment]
Hey @mitnk, thanks for cicada. I really like its simplicity.
Are you sure about `libffi` being a runtime dependency? As a build dependency it's already being satisfied by rust itself as a transitive dependency (`rust -> llvm-libs -> libffi`).

**Comment by mitnk** [comment]
Ah, let me try again with a clean environment.

**Comment by mitnk** [comment]
Just tried with latest archlinux docker image, the build just works fine. So I guess what I encountered were other issues (it's not a runtime error, it was a error during a `makepkg`).

Closing the PR. Thanks for the review!
